### ISSUE TITLE
require-traverser is causing parsing of package.json by acorn

### ISSUE THREAD

**Issue by BryanCrotaz** [fresheneesz/requireTraverser#1]
In requireTraverser.js

```
var traverse = function(dependencies, dependencyMap, opts) {
    var mainFutures = []
	dependencies.forEach(function(dependencyFile) {
        // ADD .....
        if (dependencyFile.endsWith(".json"))
        {
            return;
        }
        ...
```

**Comment by fresheneesz** [comment]
@BryanCrotaz Thanks for bringing this up! Mind creating a pull request? Perhaps with a unit test?

**Comment by BryanCrotaz** [comment]
PR #2

**Comment by fresheneesz** [comment]
Thanks for pointing this out and fixing it! Published it to version 0.1.9. Btw, feel free to star this repo if you like it ; )
